config/api-postman.php

```diff
@@ -45,4 +45,23 @@
         ],
     ],
 
+    /*
+     * Enable Form Data.
+     *
+     * Determines whether or not form data should be handled.
+     */
+
+    'enable_formdata' => false,
+
+    /*
+     * Form Data.
+     *
+     * The key/values to requests for form data dummy information.
+     */
+
+    'formdata' => [
+        'email' => 'john@example.com',
+        'password' => 'changeme',
+    ],
```

Comment:
What is the purpose of this? Is this hard coding values?

Reply:
Dummy data for the generated collections, e.g. anything with "email" and "password" will be populated as so, this means you can just easily go in and start sending requests in postman without having to populate data manually.

Reply:
To be discussed further